hail/python/hailtop/gear/auth/hailjwt.py

```diff
@@ -107,20 +108,20 @@ def parse_header(request):
 
 
 def rest_authenticated_users_only(fun):
-    token_getter = lambda request: parse_header(request)
-    return authenticated_users_only(token_getter)(fun)
+    return authenticated_users_only(parse_header)(fun)
 
 
 def rest_authenticated_developers_only(fun):
-    token_getter = lambda request: parse_header(request)
-    return authenticated_developers_only(token_getter)(fun)
+    return authenticated_developers_only(parse_header)(fun)
+
```

Comment:
I can't figure out why this file and the one below still show up as changes. I checked them out from master to try and revert my changes...